python/ELASTICSEARCH_ENCRYPTED_AT_REST/ELASTICSEARCH_ENCRYPTED_AT_REST.py

```diff
@@ -0,0 +1,348 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+"""
+#####################################
+##           Gherkin               ##
+#####################################
+
+Rule Name:
+    ELASTICSEARCH_ENCRYPTED_AT_REST
+Description:
+    Checks whether Amazon Elasticsearch Service (Amazon ES) domains have encryption at rest configuration enabled. The resource is NON_COMPLIANT if encryption at rest is not enabled.
+Trigger:
+    Periodic
+Reports on:
+    AWS::Elasticsearch::Domain
+Rule Parameters:
+    None
+
+Scenarios:
+  Scenario: 1
+     Given: No Amazon Elasticsearch Service (Amazon ES) domains are present in a region for an account.
+     Then: Return NOT_APPLICABLE
+  Scenario: 2
+     Given: At least one Elasticsearch Service Domain is present
+     Given: Enabled field is true in EncryptionAtRestOptions
+     Then: Return COMPLIANT
+  Scenario: 3
+     Given: At least one Elasticsearch Service Domain is present
+     Given: Enabled field is false in EncryptionAtRestOptions
+     Then: Return NON_COMPLIANT
+"""
+
+import json
+import sys
+import datetime
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::Elasticsearch::Domain'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    es_client = get_client('es', event)
+    list_domain = es_client.list_domain_names()
+    evaluations = []
+    if not list_domain['DomainNames']:
+        return 'NOT_APPLICABLE'
+    domain_list = []
+    for domain in list_domain['DomainNames']:
+        domain_list.append(domain['DomainName'])
+
+    domain_config = es_client.describe_elasticsearch_domains(DomainNames=domain_list)
+    for domain in domain_config['DomainStatusList']:
+        if domain['EncryptionAtRestOptions']['Enabled']:
+            evaluations.append(build_evaluation(domain['DomainName'], 'COMPLIANT', event))
+        else:
+            evaluations.append(build_evaluation(domain['DomainName'], 'NON_COMPLIANT', event, annotation='The Amazon Elasticsearch domain does not have the encryption of data at rest as enabled'))

```

Comment:
Correct to `This Amazon Elasticsearch domain does not have encryption of data at rest enabled.`